src/main/java/appeng/client/gui/implementations/GuiAmount.java

```diff
@@ -150,7 +153,12 @@ protected int addOrderAmount(final int i) {
 
     protected int getAmount() {
         String out = this.amountTextField.getText();
-        double resultD = Calculator.conversion(out);
+        double resultD;
+        if (Loader.isModLoaded("gtnhlib")) {
```

Comment:
What about caching this condition instead of calling `Loader.isModLoaded` every time?

Reply:
Also what about changing implementation of `Calculator.conversion` to automatically call GTNHLib when possible? (Instead of changing every *usage* of this method).

Reply:
I had *some* reasons for doing it this way, but I admit those were pretty bad reasons. Changing `Calculator` itself also means I can check for (and actually cache the presence of) GTNHLib in one place.

Thanks for the feedback!